src/Build/Evaluation/Expander.cs

```diff
@@ -3352,6 +3352,12 @@ internal object Execute(object objectInstance, IPropertyProvider<T> properties,
                     }
                     else
                     {
+                        // Check that the function that we're going to call is valid to call
+                        if (!IsInstanceMethodAvailable(_methodMethodName))
+                        {
+                            ProjectErrorUtilities.ThrowInvalidProject(elementLocation, "InvalidFunctionMethodUnavailable", _methodMethodName, _receiverType.FullName);
```

Comment:
Should we modify the message to mention that there's a way to enable all methods with  `Traits.Instance.EnableAllPropertyFunctions`? It would be less painful for customers this way.